contrib/bindings/go/pathrs/capi.go

```diff
@@ -0,0 +1,178 @@
+// Copyright (C) 2019, 2020 Aleksa Sarai <cyphar@cyphar.com>
+// Copyright (C) 2019, 2020 SUSE LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package pathrs
+
+/*
+#cgo LDFLAGS:-L ../../../../target/debug/ -l pathrs
+#include "../../../../include/pathrs.h"
+*/
+import "C"
+import (
+	"unsafe"
+)
+
+type pointerType = int
+
+const (
+	pointerOnNone   pointerType = C.PATHRS_NONE
```

Comment:
I don't really like these names ("pointer on ..." isn't proper English), but I can't think of anything nicer than `pointerToNone` or `ptrTypeNone`.